src/main/scala/mesosphere/marathon/api/serialization/ContainerSerializer.scala

```diff
@@ -2,38 +2,61 @@ package mesosphere.marathon.api.serialization
 
 import mesosphere.marathon.Protos
 import mesosphere.marathon.core.externalvolume.ExternalVolumes
-import mesosphere.marathon.state.Container.Docker
 import mesosphere.marathon.state.Container.Docker.PortMapping
 import mesosphere.marathon.state._
-import org.apache.mesos
+import org.apache.mesos.{ Protos => Mesos }
```

Comment:
I really would have liked all these import renames in a separate commit, as I am reluctant to merge those. Renaming an import is something scala allows us to, in order to avoid namespace conflicts, but we shouldn't do that if we don't really need to.

1. imo simply importing `org.apache.mesos` and using `mesos.Protos` is the obvious choice and clearer than renaming `mesos.Protos => Mesos`.
2. Splitting up the marshaller layers (`model <-> mesos.Protos` for communication to Mesos; `model <-> marathon.Protos` for persisting objects in Marathon) would have made import renames completely unneeded as each Protos object is unambiguous in its own scope.

I know reverting the renames or changing them is tedious as the IDE doesn't help you, but they're not needed to merge the required changes and I'd rather keep the change set small – unless the extra changes add value by separating concerns as described in 2).